core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -136,6 +136,31 @@ describe('Learner dashboard functionality', function() {
     users.logout();
   });
 
+  it('should add exploration to play later list', function() {
+    var EXPLORATION_FRACTION = 'fraction';
+    var CATEGORY_MATHEMATICS = 'Mathematics';
+    var LANGUAGE_ENGLISH = 'English';
+    var EXPLORATION_OBJECTIVE = 'hold the light of two trees';
+
+    users.createUser(
+      'creator@learnerDashboard.com', 'creatorLearnerDashboard');
+    users.login('creator@learnerDashboard.com');
+    workflow.createAndPublishExploration(
+      EXPLORATION_FRACTION, CATEGORY_MATHEMATICS,
+      EXPLORATION_OBJECTIVE, LANGUAGE_ENGLISH);
```

Comment:
Let's add one more exploration and check a specific exploration can be added in the play later list.

Reply:
Done